### PR TITLE
LPS-161200 [Raylife AP] Adjustment Status chart

### PR DESCRIPTION
**Pull request by silasbsi**
Subtask: [LPS-161200](https://issues.liferay.com/browse/LPS-161200)

### CODE REVIEW

**Comment by silasbsi** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/services/Application.js [hunk 1]
```diff
@@ -20,6 +20,12 @@ export function getApplicationsStatus(status) {
 	return axios.get(`${DeliveryAPI}/?filter=applicationStatus eq '${status}'`);
 }
 
+export function getApplicationsStatusTotal() {
+	return axios.get(
+		`${DeliveryAPI}/?aggregationTerms=applicationStatus&filter`
+	);
+}
+
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/services/Application.js:28]:
```suggestion
export function getApplicationsStatusTotal() {
	return axios.get(
		`${DeliveryAPI}/?aggregationTerms=applicationStatus`
	);
}

```

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job! I added some suggestions :)

**Review by joanagrego — APPROVED** [review]
Nice job! :white_check_mark:

**Comment by silasbsi** [comment]
ci:test:sf

**Review by eliasantosf — APPROVED** [review]
LGTM :fire: Thanks @silasbsi

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**